nilearn/_utils/testing.py

```diff
@@ -96,7 +102,9 @@ def write_tmp_imgs(*imgs, **kwargs):
         with warnings.catch_warnings():
             warnings.simplefilter("ignore", RuntimeWarning)
             for img in imgs:
-                filename = os.tempnam(None, "nilearn_") + ".nii"
+                (_, filename) = tempfile.mkstemp(prefix="nilearn_",
```

Comment:
good point.  since only one argument is being used, how do you feel about removing the output argument list and just subscripting the function call: 
`filename = tempfile.mkstemp(prefix="nilearn_", suffix=".nii", dir=None)[1]`